pkg/docker/containers-view.jsx

```diff
@@ -308,6 +309,50 @@ var ImageDetails = React.createClass({
     }
 });
 
+var ImageSecurity = React.createClass({
+    render: function () {
+        var info = this.props.info;
+        var text, rows;
+        var args = {
+            time: info.finishedTime.format('MMM Do'),
+            type: info.scanType,
+            count: info.vulnerabilities.length
+        };
+
+        if (info.successful === false) {
+            text = cockpit.format(_('The scan from $time ($type) was not successful.'), args);
+
+        } else if (args.count === 0) {
+            text = cockpit.format(_('The scan from $time ($type) found no vulnerabilities.'), args);
+
+        } else {
+            text = cockpit.ngettext(cockpit.format('The scan from $time ($type) found one vulnerability:', args),
+                                    cockpit.format('The scan from $time ($type) found $count vulnerabilities:', args),
```

Comment:
This needs to be the other way around. The arguments to ngettext should be the strings, and ```cockpit.format``` should operate on the result. At least that's how I understand it. If this does actually work, both the scanning and translating, then let me know.